internal/services/oracledatabase/exadata_infrastructure_data_source.go

```diff
@@ -0,0 +1,574 @@
+						AdditionalStorageCount:      pointer.From(prop.AdditionalStorageCount),
+						AvailableStorageSizeInGbs:   pointer.From(prop.AvailableStorageSizeInGbs),
+						CpuCount:                    pointer.From(prop.CpuCount),
+						ComputeCount:                pointer.From(prop.ComputeCount),
+						CustomerContacts:            FlattenCustomerContacts(prop.CustomerContacts),
+						DataStorageSizeInTbs:        pointer.From(prop.DataStorageSizeInTbs),
+						DbNodeStorageSizeInGbs:      pointer.From(prop.DbNodeStorageSizeInGbs),
+						DbServerVersion:             pointer.From(prop.DbServerVersion),
+						DisplayName:                 prop.DisplayName,
+						EstimatedPatchingTime:       FlattenEstimatedPatchingTimes(prop.EstimatedPatchingTime),
+						LastMaintenanceRunId:        pointer.From(prop.LastMaintenanceRunId),
+						LifecycleDetails:            pointer.From(prop.LifecycleDetails),
+						LifecycleState:              string(*prop.LifecycleState),
+						MaintenanceWindow:           FlattenMaintenanceWindow(prop.MaintenanceWindow),
+						MaxCPUCount:                 pointer.From(prop.MaxCPUCount),
+						MaxDataStorageInTbs:         pointer.From(prop.MaxDataStorageInTbs),
+						MaxDbNodeStorageSizeInGbs:   pointer.From(prop.MaxDbNodeStorageSizeInGbs),
+						MaxMemoryInGbs:              pointer.From(prop.MaxMemoryInGbs),
+						MemorySizeInGbs:             pointer.From(prop.MemorySizeInGbs),
+						MonthlyDbServerVersion:      pointer.From(prop.MonthlyDbServerVersion),
+						MonthlyStorageServerVersion: pointer.From(prop.MonthlyStorageServerVersion),
+						NextMaintenanceRunId:        pointer.From(prop.NextMaintenanceRunId),
+						OciUrl:                      pointer.From(prop.OciUrl),
+						Ocid:                        pointer.From(prop.Ocid),
+						ProvisioningState:           string(*prop.ProvisioningState),
+						Shape:                       prop.Shape,
+						StorageCount:                pointer.From(prop.StorageCount),
+						StorageServerVersion:        pointer.From(prop.StorageServerVersion),
+						TimeCreated:                 pointer.From(prop.TimeCreated),
+						TotalStorageSizeInGbs:       pointer.From(prop.TotalStorageSizeInGbs),
+					}
+				}
+
+				systemData := model.SystemData
+				if systemData != nil {
+					output.SystemData = []SystemDataModel{
+						{
+							CreatedBy:          systemData.CreatedBy,
+							CreatedByType:      systemData.CreatedByType,
+							CreatedAt:          systemData.CreatedAt,
+							LastModifiedBy:     systemData.LastModifiedBy,
+							LastModifiedbyType: systemData.LastModifiedbyType,
+							LastModifiedAt:     systemData.LastModifiedAt,
+						},
+					}
+				}
+				output.Name = id.CloudExadataInfrastructureName
+				output.ResourceGroupName = id.ResourceGroupName
+				output.Type = pointer.From(model.Type)
+				output.Tags = utils.FlattenPtrMapStringString(model.Tags)
+				output.Location = model.Location
+				output.Zones = model.Zones
```

Comment:
why are these seperate and not included above?

Reply:
I've moved them up with the test.